src/System Application/App/File System/src/Account/FileAccounts.Page.al

```diff
@@ -0,0 +1,322 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.FileSystem;
+
+using System.Telemetry;
+
+/// <summary>
+/// Lists all of the registered file accounts
+/// </summary>
+page 9450 "File Accounts"
+{
+    PageType = List;
+    Caption = 'File Accounts';
+    ApplicationArea = All;
+    UsageCategory = Administration;
+    SourceTable = "File Account";
+    SourceTableTemporary = true;
+    InsertAllowed = false;
+    ModifyAllowed = false;
+    DeleteAllowed = false;
+    Editable = false;
+    ShowFilter = false;
+    LinksAllowed = false;
+    RefreshOnActivate = true;
+    InherentPermissions = X;
+    InherentEntitlements = X;
+
+    layout
+    {
+        area(Content)
+        {
+            repeater(Accounts)
+            {
+                FreezeColumn = NameField;
+                field(LogoField; Rec.Logo)
+                {
+                    ShowCaption = false;
+                    Caption = ' ';
+                    ToolTip = 'Specifies the logo for the type of file account.';
+                    Width = 1;
+                }
+                field(NameField; Rec.Name)
+                {
+                    ToolTip = 'Specifies the name of the account.';
+                    Visible = not IsInLookupMode;
+
+                    trigger OnDrillDown()
+                    begin
+                        ShowAccountInformation();
+                    end;
+                }
+                field(NameFieldLookup; Rec.Name)
+                {
+                    ToolTip = 'Specifies the name of the account.';
+                    Visible = IsInLookupMode;
+                }
+                field(DefaultField; DefaultTxt)
+                {
+                    Caption = 'Default';
+                    ToolTip = 'Specifies whether the file account will be used for all scenarios for which an account is not specified. You must have a default file account, even if you have only one account.';
+                    Visible = not IsInLookupMode;
+                }
+                field(FileConnector; Rec.Connector)
+                {
+                    ToolTip = 'Specifies the type of file extension that the account is added to.';
+                    Visible = false;
+                }
+            }
+        }
+
+        area(FactBoxes)
+        {
+            part(Scenarios; "File Scenarios FactBox")
+            {
+                Caption = 'File Scenarios';
+                SubPageLink = "Account Id" = field("Account Id"), Connector = field(Connector), Scenario = filter(<> 0); // Do not show Default scenario
```

Comment:
change to `SubPageLink = "Account Id" = field("Account Id"), Connector = field(Connector), Scenario = filter(<> Default);` and remove comment